doc/developer/testdrive.md

```diff
@@ -730,6 +730,11 @@ Send the data to the specified partition.
 
 Obtains the data from the specified `sink` and compares it to the expected data recorded in the test. The comparison algorithm is sensitive to the order in which data arrives, so `sort-messages=true` can be used along with manually pre-sorting the expected data in the test. If `partial-search=usize` is specified, up to `partial-search` records will be read from the given topic and compared to the provided records. The recordsdo not have to match starting at the beginning of the sink but once one record matches, the following must all match.  There are permitted to be records remaining in the topic after the matching is complete.  Note that if the topic is not required to have `partial-search` elements in it but there will be an attempt to read up to this number with a blocking read.
 
+#### Headers
+
+Additionally, the first lines of data passed into `kafka-ingest`, if they start with `header:`, accepts a
+`<key text>=<value text>` format for adding kafka headers to the messages sent through kafka.
```

Comment:
Quibble on this: if the `headers` are going to apply to the `kafka-ingest` command as a whole (rather than an individual row), then they should be passed via an argument to the action! I'll try to put up a diff to show you what I mean!

Reply:
testdrive has a weird special case for JSON-formatted maps as parameter values...

```
$ kafka-ingest headers={"a": "b"}
``

...so you can just parse as a [`serde_json::Map`](https://docs.rs/serde_json/latest/serde_json/struct.Map.html) and I think it will all check out.